source/portal/v11Client.js

```diff
@@ -165,13 +179,22 @@ var V11Client = function(clientId, sigConnection, portal) {
         return safeCall(callback, 'error', 'Illegal request');
       }
 
-      var subscription_id = Math.round(Math.random() * 1000000000000000000) + '';
+      var subscription_id = uuid().replace(/-/g,'');
+      let transport_id;
       return validateSubReq(subReq)
         .then((req) => {
-          req.type = 'webrtc';//FIXME: For backend compatibility with v3.4 clients.
+          if (req.transport && req.transport.type == 'quic') {
+            req.type = 'quic';
+            if (!req.transport.id) {
+              req.transport.id = uuidWithoutDash();
+            }
+            transport_id = req.transport.id;
+          } else {
+            req.type = 'webrtc';//FIXME: For backend compatibility with v3.4 clients.
+          }
           return portal.subscribe(clientId, subscription_id, req);
         }).then((result) => {
-          safeCall(callback, 'ok', {id: subscription_id});
+          safeCall(callback, 'ok', {id: subscription_id, transportId: transport_id});
```

Comment:
spec says subscriptionId, you're using id here.

Reply:
Also in spec please specify "transportId" to be undefined if it's response for webrtc.

Reply:
It looks like this mismatch is not introduced by this PR. I also find it in master branch. File a bug for it?